### ISSUE TITLE
Player foley archive 18235e0c9ec0e636 fails to load

### ISSUE THREAD

**Issue by RaidingForPants** [RaidingForPants/hd2-audio-modder#4]
```
Traceback (most recent call last):
  File "tkinter_init.py", line 1921, in call_
  File "audio_modder.py", line 2443, in load_archive
  File "audio_modder.py", line 1715, in load_archive_file
  File "audio_modder.py", line 1010, in from_file
  File "audio_modder.py", line 1212, in load
KeyError: 422952040 
```

Issue is caused by one soundbank in the archive containing the hierarchy information for sound 422952040, while a different soundbank contains the actual audio data. Since the program loads one bank at a time, it is thus not able to find the audio data for the sound when it is parsing the soundbank hierarchy of the first bank. May have to parse all banks before creating the associated AudioSources.

**Comment by RaidingForPants** [comment]
Only the content/audio/foley_player soundbank has a hierarchy. All the other banks (e.g. foley_player_surface_media_rubber) only have audio data, no hierarchy. And the foley_player soundbank has ids from the other soundbanks in its hierarchy.

**Comment by RaidingForPants** [comment]
Fixed in v1.13.1
